worker/provisioner/container_initialisation.go

```diff
@@ -265,20 +265,18 @@ func (cs *ContainerSetup) getContainerArtifacts(
 		return nil, nil, nil, err
 	}
 
-	bridger, err := network.DefaultEtcNetworkInterfacesBridger(activateBridgesTimeout, instancecfg.DefaultBridgePrefix, systemNetworkInterfacesFile)
-	if err != nil {
-		return nil, nil, nil, err
-	}
-
 	switch containerType {
 	case instance.KVM:
 		initialiser = kvm.NewContainerInitialiser()
-		broker, err = NewKvmBroker(
-			bridger,
-			cs.machine.MachineTag(),
+		manager, err := kvm.NewContainerManager(managerConfig)
+		if err != nil {
+			return nil, nil, nil, err
+		}
+		broker, err = NewKVMBroker(
+			cs.prepareHost,
 			cs.provisioner,
+			manager,
```

Comment:
'NewKvmBroker' (lowercase) took a manager config, 'NewKVMBroker' always took a 'manager' rather than the config to match the LXD broker code.